demo/github/4-submit.sh

```diff
@@ -5,12 +5,10 @@
 set -ex
 
 SCITT_URL=${SCITT_URL:-"https://127.0.0.1:8000"}
-SCITT_TRUST_STORE=tmp/trust_store
 
 TMP_DIR=tmp/github
 
 scitt submit $TMP_DIR/claims.cose \
     --receipt $TMP_DIR/claims.receipt.cbor \
     --url "$SCITT_URL" \
-    --service-trust-store $SCITT_TRUST_STORE \
```

Comment:
Purely because it does not make much sense in verifying in a submit command and the argument is confusing in its name of why it is necessary. There is a separate `validate` command for this.

Reply:
Sounds good to me. Then it probably makes sense to directly remove the corresponding CLI argument from the `submit` command so that we avoid users to get confused with its usage.

Reply:
argument removed from submit command as well